src/Routing/Router.php

```diff
@@ -206,9 +206,12 @@ public static function getNamedExpressions(): array
      * @throws \Cake\Core\Exception\CakeException
      * @see \Cake\Routing\RouteBuilder::connect()
      * @see \Cake\Routing\Router::scope()
+     * @deprecated 4.3.0 Use the `RouterBuilder`'s `connect()` method instead.
```

Comment:
```suggestion
     * @deprecated 4.3.0 Use `RouterBuilder::connect()` instead.
```
Hopefully simpler english.